language/stdlib/modules/LibraTimestamp.move

```diff
@@ -151,7 +153,7 @@ module LibraTimestamp {
     }
 
 
-    /// Gets the timestamp representing `now` in microseconds.
+    /// Gets the current time in microseconds.
     public fun now_microseconds(): u64 acquires CurrentTimeMicroseconds {
```

Comment:
More of a meta question here: now that we have `now_seconds` do we want to keep `now_microseconds` around as part f the public API since we don't need that fine of a granularity on time?

Reply:
Great question! I don't see `now_microseconds` used anywhere else except for `AccountLimits`, and I would be very much in favor of replacing that usage with `now_seconds` and hiding `now_microseconds`. Let me try to tackle this before landing.

Reply:
Starting looking at this and it feels like a big enough change that it should happen in a separate PR. But definitely worth doing!